modules/fedimint-ln-client/src/lib.rs

```diff
@@ -236,18 +239,14 @@ impl LightningClientExt for Client {
             )
             .await?;
 
-        let mut dbtx = self.db().begin_transaction().await;
-        self.add_operation_log_entry(
-            &mut dbtx,
-            operation_id,
-            LightningCommonGen::KIND.as_str(),
-            LightningMeta::Receive {
-                out_point: OutPoint { txid, out_idx: 0 },
-                invoice: invoice.clone(),
-            },
-        )
-        .await;
-        dbtx.commit_tx().await;
+        // Wait for the transaction to be accepted by the federation, otherwise the
+        // invoice will not be able to be paid
+        //self.transaction_updates(operation_id)
+        //    .await
+        //    .await_tx_accepted(txid)
+        //    .await
+        //    .map_err(|e| anyhow::anyhow!("Offer transaction was not accepted: {:?}",
+        // e))?;
```

Comment:
If you comment that in again you can get rid of the while loop in `ClientNg::LnInvoice` as mentioned above.

Reply:
Oops! Apologies for that, meant to do the fix you mentioned :)